zebra-rpc/src/methods/tests/snapshots/get_blockchain_info@mainnet_10.snap

```diff
@@ -86,6 +86,11 @@ expression: info
       "name": "NU6",
       "activationheight": 2726400,
       "status": "pending"
+    },
+    "ffffffff": {
```

Comment:
Will we remember to update this placeholder hex when the time comes?

Reply:
As I am aware the snap files are auto-generated. Once the code is updated, this should be too.